controllers/pwa.js

```diff
@@ -22,51 +22,46 @@ const lighthouseLib = require('../lib/lighthouse');
 const Pwa = require('../models/pwa');
 const router = express.Router(); // eslint-disable-line new-cap
 const libMetadata = require('../lib/metadata');
+const cache = require('../lib/data-cache');
 
 const LIST_PAGE_SIZE = 32;
 const DEFAULT_PAGE_NUMBER = 1;
 const DEFAULT_SORT_ORDER = 'newest';
 
+const CACHE_LIFETIME = 60 * 60 * 24; // 24 hours
```

Comment:
To check if the page has changed, we're rendering again and comparing before updating the cache. So, for every hit, we're loading data from the database again. This is not a problem now, but may create a performance issue if we get a traffic spike. What happens if we get to 50 requests / second?